index.html

```diff
@@ -289,6 +289,93 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <footer>
+        <section class="contact">
+          <form id="form" action="https://formspree.io/f/mlezobkj" method="POST">
+            <div id="contact-section" class="container-contact">
+              <p class="description-contact">
+                I'm always interested in hearing about new projects, so if you'd
+                like to chat please get in touch.
+              </p>
+              <ul class="contact-inputs">
+                <li class="form-field">
+                  <input
+                    id="full-name"
+                    name="full-name"
+                    type="text"
+                    maxlength="30"
+                    placeholder="Full Name"
+                    required
+                  />
+                </li>
+                <li class="form-field">
+                  <input
+                    id="email-address"
+                    name="email-address"
+                    type="email"
+                    placeholder="Email address"
+                    required
+                  />
+                  <small></small>
+                </li>
+                <li class="form-field">
+                  <textarea
+                    id="text-area"
+                    name="text-area"
+                    maxlength="500"
+                    placeholder="Enter your text here"
+                    required
+                  ></textarea>
+                </li>
+              </ul>
+            </div>
+            <div class="container-button">
+              <button class="button-submit" type="submit">Get in touch</button>
+            </div>
+          </form>
+        </section>
+        <hr class="top-line-social-icons" />
+        <div class="container-icons-footer">
+          <a target="_blank" href="https://github.com/BregornOriginal">
+            <img
+              class="social-icons-footer"
+              src="./media/svg/github.svg"
+              alt="Git-Hub-icon-Footer"
+            />
+          </a>
```

Comment:
Great job fixing the src attribute of your icons! ✅ However, at the moment they have a very low color contrast, and people with reduced vision and other visual conditions might have some problems perceiving them. In order to fix this and make your portfolio more accessible to everyone, you can use high-contrast colors. 🎨  Remember to stick as much as possible to the Figma design, kindly fix this: 🔧 

You can read further about this concept [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast).

Your design:
![image](https://user-images.githubusercontent.com/77069011/197205806-cf8cf13b-3abb-411f-8339-02ec4a73bcce.png)

Figma template:
![image](https://user-images.githubusercontent.com/77069011/197205852-d1fe3d13-92a8-4ec3-945f-92c35f862605.png)